packages/terra-core-docs/CHANGELOG.md

```diff
@@ -4,10 +4,13 @@
 
 * Changed
   * linter fixes.
+  * Terra Icon accessibility guide.
```

Comment:
is this redundant?

Reply:
Removed in b9e422e34d1181a337047253bf1dda4e510c656a